corehq/apps/users/models.py

```diff
@@ -3195,3 +3195,12 @@ def check_and_send_limit_email(domain, plan_limit, user_count, prev_count):
         }),
     )
     return
+
+
+class ConnectIDUserLink(models.Model):
+    connectid_username = models.TextField()
+    commcare_user = models.ForeignKey(User, related_name='connectid_user', on_delete=models.CASCADE)
```

Comment:
Just noting that I was initially confused by the `commcare_user` name since it generally refers to the Couch user in the rest of the codebase. I do see the issue with calling it `user` since there are 2 user's linked to the model. 🤷🏻‍♂️